spotbugs-annotations/build.gradle

```diff
@@ -86,5 +86,5 @@ apply from: "$rootDir/gradle/jigsaw.gradle"
 spotbugs {
   effort = "max"
   reportLevel = "high"
-  toolVersion '3.1.0-RC6'
+  toolVersion '3.1.0-RC7'
```

Comment:
Why do we need those versions here at all? I'm not the gradle expert, but can't we somehow say "use the latest"?

Reply:
We should at least be able to use a global variable (e.g. `ext.spotbugsVersion`) in the root build.gradle

Reply:
or delete `toolVersion` and update version of gradle version like https://github.com/spotbugs/spotbugs/pull/452/files

then it will use RC7.

Reply:
I'm +1 on the solution which will remove duplicated version mentions all around, but my Gradle knowledge is too bad to propose or to prefer any one.

Reply:
This is a "proper" subproject, right?  If I understand what you're going for, I think you can do this little bit of foo to always depend on the current build instead of using version numbers:

```
dependencies {
  spotbugs project(':spotbugs')
}

spotbugs {
  toolVersion '3.1.0' // overridden by explicit spotbugs configuration dependency
}
```

Reply:
But regardless, RC6 is obviously wrong, so this patch should go into master, no?

Reply:
I silently hoped that someone would have a silver bullet to get rid of this versions once and forever :-)

Reply:
Looks like @KengoTODA just found the right bullet, see #451.
Kendo: *this* PR is not needed then, right?

Reply:
Cool. Does it take filter.rst as well?

Reply:
There were still five RC6 entries after #451 got merged, I've updated my commit to match that.